cleansio/censor/censor.py

```diff
@@ -12,13 +13,28 @@ def __init__(self, explicits):
     def censor_audio_chunk(self, file_path):
         """ Common process to censor an audio chunk """
         audio_segment = AudioSegment.from_file(file_path)
+        audio_segment = self.__band_pass_filter(audio_segment)
```

Comment:
Like I said in person it has to modify the -accuracy chunks. You would have to create a new AudioSegment object from file_path + '-accuracy'